eq-author/src/App/page/Design/CalculatedSummaryPageEditor/index.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { propType } from "graphql-anywhere";
+import { get, flowRight } from "lodash";
+import styled from "styled-components";
+import gql from "graphql-tag";
+
+import { colors } from "constants/theme";
+import RichTextEditor from "components/RichTextEditor";
+import withChangeUpdate from "enhancers/withChangeUpdate";
+import withUpdateCalculatedSummaryPage from "./withUpdateCalculatedSummaryPage";
+import withEntityEditor from "components/withEntityEditor";
+import PageHeader from "../PageHeader";
+
+import withPropRenamed from "enhancers/withPropRenamed";
+import AnswerSelector from "./AnswerSelector";
+import {
+  ANSWER,
+  METADATA,
+  VARIABLES,
+} from "components/ContentPickerSelect/content-types";
+
+const titleControls = {
+  emphasis: true,
+  piping: true,
+};
+
+const PageSegment = styled.div`
+  padding: 0 2em;
+`;
+
+const SelectorTitle = styled.h2`
+  font-size: 1em;
+  color: ${colors.darkGrey};
+`;
+
+export class CalculatedSummaryPageEditor extends React.Component {
```

Comment:
This could be a functional component as it implements render. This could possibly be a lint.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md